Appraisals

```diff
@@ -8,4 +8,11 @@ end
 appraise 'rails-6' do
   gem 'rails', '~> 6.0.0'
   gem 'mysql2', '~> 0.5'
+  gem "net-smtp", require: false
+end
+
+appraise 'rails-61' do
```

Comment:
For convention can we change this to `6-1`? I think that’s generally more clear for version mapping.

Reply:
Done